CONTRIBUTING.rst

```diff
@@ -17,15 +19,12 @@ Before any pull request can be accepted, you must do the following:
 * Add your Github username (alphabetically) to the in-repository list that we use
   to track CLA signatures:
   `tools/.github-cla-signers`_
-* Add or update any `unit tests`_ accordingly
-* Add or update any `integration tests`_ (if applicable)
+* Add or update any :ref:`unit tests<testing>` accordingly
+* Add or update any :ref:`integration tests` (if applicable)
```

Comment:
integration tests needs and underscore instead of to find the right anchor.
```suggestion
* Add or update any :ref:`integration_tests` (if applicable)
```